float8_experimental/float8_ops.py

```diff
@@ -76,6 +84,7 @@ def float8_cat(aten_op, args, kwargs=None):
     scale = chunked_tensors[0]._scale
```

Comment:
Future PR:
We should code share between unflatten/flatten and here to just splat out the extra metadata that lives on the tensor